src/MSCorEE.Desktop/MSCorEE.Helpers+IClrMetaHost.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Runtime.InteropServices;
+    using System.Text;
+
+    /// <content>
+    /// Methods and nested types that are not strictly P/Invokes but provide
+    /// a slightly higher level of functionality to ease calling into native code.
+    /// </content>
+    public static partial class MSCorEE
+    {
+        private static readonly Guid CLSID_CLRMetaHost = new Guid("{9280188d-0e8e-4867-b30c-7fa83884e8de}");
+        private static readonly IClrMetaHost ClrMetaHost = CreateClrMetaHost();
+
+        public static IEnumerable<string> GetProcessRuntimes(SafeHandle handle)
```

Comment:
I don't think any of these methods belong in PInvoke, per our contributing.md doc.

Please review https://github.com/AArnott/pinvoke/blob/master/CONTRIBUTING.md#helper-methods